src/backend/app.js

```diff
@@ -19,12 +21,89 @@ app.use(express.json());
 
 app.use(cors());
 
+router.use("/reservations", reservationsRouter);
 router.use("/meals", mealsRouter);
 
+//Respond with all meals in the future (relative to the when datetime) - /future-meals
+
+app.get("/future-meals", async (req, res) => {
+  try {
+    const today = new Date();
+    const futureMeals = await knex("meal").where("event_date", ">", today);
+    if (futureMeals.length === 0) {
+      res.status(204).json(futureMeals);
+    }
+    res.status(200).json(futureMeals);
+  } catch (error) {
+    console.error(error);
+    res.status(500).send("Server Error");
+  }
+});
+
+//Respond with all meals in the past (relative to the when datetime) - /past-meals
+
+app.get("/past-meals", async (req, res) => {
+  try {
+    const today = new Date();
+    const pastMeals = await knex("meal").where("event_date", "<", today);
+    if (pastMeals.length === 0) {
+      res.status(204).json(pastMeals);
+    }
+    res.status(200).json(pastMeals);
+  } catch (error) {
+    console.error(error);
+    res.status(500).send("Server Error");
+  }
+});
+
+//Respond with all meals sorted by ID - /all-meals
+
+app.get("/all-meals", async (req, res) => {
```

Comment:
isn't that the same as what `/meals` called with `GET` returns? I'm not sure if we need two endpoints that return exactly the same thing :)